pkgs/os-specific/linux/compsize/default.nix

```diff
@@ -13,18 +13,17 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ btrfs-progs ];
 
-  installPhase = ''
-    mkdir -p $out/bin
+  PREFIX = placeholder "out";
+
+  preInstall = ''
     mkdir -p $out/share/man/man8
-    install -m 0755 compsize $out/bin
-    install -m 0444 compsize.8 $out/share/man/man8
   '';
 
   meta = with lib; {
     description = "btrfs: Find compression type/ratio on a file or set of files";
-    homepage    = "https://github.com/kilobyte/compsize";
-    license     = licenses.gpl2Plus;
+    homepage = "https://github.com/kilobyte/compsize";
```

Comment:
This is a good cleanup, but perhaps we should eliminate the formatting with a force push since eliminating format related merge conflicts is an explicite goal of nixpkgs-fmt.

Reply:
This is fine.